cfpb/consumerfinance.gov
# Switch oah handlebars templates to string constants and templates

We include handlebars in here for purely historical reasons. This removes our handlebars templates from owning-a-home (the only place they exist) and excises their dependencies.

Testing:
 - Fire up http://localhost:8000/owning-a-home/explore-rates/
 - Mouse over the chart to check the chart templates.
 - Enter a $1111111 for the house price
 - Change the "Loan type" to check the warning messages
 - Select a county to see that county options are populated correctly
